### PR TITLE
Init proj DayOfWeek

### PR DESCRIPTION
**Pull request by ikazlou**
1. Git
2. Screen
image
3. 11.06.2020 / 13.06.2020
4. Norm

### CODE REVIEW

**Comment by MikhailMasny** [comment]
1. incorrect label, need to use another
2. incorrect commit format, should be repeated in the next lesson (remind me)
3. need screenshots of the program, not a code

File: src/Selfpreporation1/SelfPreporation1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            Wednesday,
+            Thursday,
+            Friday,
+            Saturday,
+            Sunday
+        }
+
+        static void Main(string[] args)
+        {
+            //int Monday = 1, Tuesday = 2, Wednesday = 3, Thursday = 4, Friday = 5, Saturday = 6, Sunday = 7;
```
Comment (MikhailMasny) [inline src/Selfpreporation1/SelfPreporation1/Program.cs:20]: Delete unused comments

File: src/Selfpreporation1/SelfPreporation1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+
+            //Console.WriteLine("You choose", 1 = Monday, 
+            
+                
+                
+                
+            //Console.WriteLine($"Hello {enum}");
+            //DayOfWeek op;
```
Comment (MikhailMasny) [inline src/Selfpreporation1/SelfPreporation1/Program.cs:65]: Delete unused comments

**Review by ikazlou — COMMENTED** [review]
Deleted comments.
Deleted inputs.
Rebild

**Pull request merged**